include/TimePos.h

```diff
@@ -57,17 +57,17 @@ class LMMS_EXPORT TimeSig
 };
 
 
-class LMMS_EXPORT MidiTime
```

Comment:
Only in case you learned what this class does: Can you please add a doxygen comment above it? (Same for `TimeSig`, also voluntarily).